cmd/eval.go

```diff
@@ -78,19 +88,26 @@ Example: combo eval -r REPLACE_ME=1,2,3 path/to/file
 				combination.WithSolveAhead(),
 			)
 
+			templateBuilder, err := template.NewBuilder(templateFile, combinations)
+			if err != nil {
+				return fmt.Errorf("failed to construct builder: %w", err)
+			}
+
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 
-			generatedFile, err := generator.Evaluate(ctx, string(file), combinations)
+			combinedTemplateDocuments, err := templateBuilder.Build(ctx)
```

Comment:
The Operator Framework typically refers to collections of yaml as manifests and we should do our best to remain consistent. Applies in all places.
```suggestion
			combinedTemplateManifests, err := templateBuilder.Build(ctx)
```